shopify-client/src/components/ShopForm.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+    <div>
```

Comment:
This div seems unnecessary

Reply:
this one is necessary because it is root div